native/src/utils/importDisplayNamesPackage.ts

```diff
@@ -0,0 +1,142 @@
+// Metro does not support dynamic imports yet: https://github.com/facebook/metro/issues/52
```

Comment:
I am watching this issue, so should it ever get resolved, I'll open an issue to replace this file but I'm not very optimistic that it'll ever happen.